packages/react-charting/src/components/assets/ErrorImage.tsx

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable @typescript-eslint/explicit-member-accessibility */
+import { ITheme } from '@fluentui/react/lib/Styling';
```

Comment:
most of the code in error and missing data image files is duplicate.

Reply:
this was due to the previous svgs which were different. Refactored when using icons.